start.sh

```diff
@@ -116,40 +118,41 @@ else
 fi
 
 cd $gaianet_base_dir
-llamaedge_wasm="$gaianet_base_dir/llama-api-server.wasm"
+llamaedge_wasm="$gaianet_base_dir/rag-api-server.wasm"
 if [ ! -f "$llamaedge_wasm" ]; then
     printf "LlamaEdge wasm not found at $llamaedge_wasm\n"
     exit 1
 fi
 
 # command to start LlamaEdge API Server
 cd $gaianet_base_dir
-cmd="wasmedge --dir .:./dashboard \
+cmd=(wasmedge --dir .:./dashboard \
   --nn-preload default:GGML:AUTO:$chat_model_name \
   --nn-preload embedding:GGML:AUTO:$embedding_model_name \
-  llama-api-server.wasm -p $prompt_type \
+  rag-api-server.wasm \
   --model-name $chat_model_stem,$embedding_model_stem \
   --ctx-size $chat_ctx_size,$embedding_ctx_size \
-  --qdrant-url http://127.0.0.1:6333 \
-  --qdrant-collection-name "default" \
-  --qdrant-limit 3 \
-  --qdrant-score-threshold 0.4 \
+  --prompt-template $prompt_type \
   --web-ui ./ \
   --socket-addr 0.0.0.0:$llamaedge_port \
   --log-prompts \
-  --log-stat"
+  --log-stat)
+
+# Add system prompt if it exists
+if [ -n "$system_prompt" ]; then
+    cmd+=("--system-prompt" "$system_prompt")
```

Comment:
This should be `rag_prompt`

Reply:
Accepted. Will rename it to `rag_prompt`. The change will be made after the next release of `rag-api-server` with the updated `--rag-prompt` CLI option. Thanks!